composer.json

```diff
@@ -19,7 +19,7 @@
     },
     "require": {
         "php": ">=5.4",
-        "cakephp/cakephp": "~3.0"
+        "cakephp/cakephp": "dev-master"
```

Comment:
It is a temporary change so that the tests pass on travis - there is no release with the stub console output yet.